modules/EasyAbp.EShop.Products/src/EasyAbp.EShop.Products.Domain.Shared/EasyAbp/EShop/Products/EShopProductsDomainSharedModule.cs

```diff
@@ -34,7 +34,7 @@ public override void ConfigureServices(ServiceConfigurationContext context)
 
             Configure<AbpExceptionLocalizationOptions>(options =>
             {
-                options.MapCodeNamespace("EasyAbp.EShop.Products", typeof(ProductsResource));
+                options.MapCodeNamespace("EasyAbp.EShop.Products.ErrorCodes", typeof(ProductsResource));
```

Comment:
I consider the `.ErrorCodes` postfix is not needed since there is also no suffix on the doc.

```json
{
  "culture": "en",
  "texts": {
    "Volo.Qa:010002": "You can not vote your own answer!"
  }
}
```